README.md

```diff
@@ -0,0 +1,149 @@
+# omigami
+#### Minimally biased features selection for untargeted metabolomics studies
+
+[![PyPI version shields.io](https://img.shields.io/pypi/v/omigami.svg)](https://pypi.python.org/pypi/omigami) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+<!-- image:: https://img.shields.io/travis/datarevenue-berlin/omigami.svg :target: https://travis-ci.org/datarevenue-berlin/omigami -->
+
+<!-- image:: https://readthedocs.org/projects/omigami/badge/?version=latest :target: https://omigami.readthedocs.io/en/latest/?badge=latest :alt: Documentation Status -->
+
+Easy and powerful tool to select features in untargeted metabolomics studies – with less risk for bias. How?
+Omigami uses multivariate models to perform a recursive feature elimination within a repeated double cross-validation.
+
+## Install
+
+```sh
+pip install omigami
+```
+
+## Motivation
+
+- It's hard to protect against selection bias on high-demensional omics data ([Krawczuk and Łukaszuk, 2016](https://www.sciencedirect.com/science/article/pii/S0933365715001426)). Even common cross-validation has been shown to overfit.
+- Most feature selection techniques focus on finding the minimal set of strongest features. Omitting redundant variables that are however still relevant to understanding the biochemical systems.
+- There's no freely available and easy-to-use Python tool that implements a minimally biased repeated double cross validation.
+- A robust selection requires many (100 - 5.000) models to be trained. Running such a large number of models in reasonable time, requires non-trivial parallelization.
+
+## Features
+
+- [x] Repeated double cross-validation
+- [x] Multivariate feature selection (Random Forest, XGB or PLS-DA)
+- [x] Minimal optimal and all relevant feature selection
+- [x] Efficient Parallelization (with Dask)
+- [x] Familiar scikit-learn API
+- [ ] Plotting
```

Comment:
we should check this, is generic enough

Reply:
I'm not sure I understand. What should we do?

Reply:
put an X between the brackets, we support some plots already in the release branch